src/main/scala/com/ambiata/poacher/hdfs/Hdfs.scala

```diff
@@ -234,7 +234,8 @@ object Hdfs {
     })
 
   def mv(src: Path, dest: Path): Hdfs[Path] = for {
-    fs <- filesystem
+    fs      <- filesystem
+    _       <- mkdir(dest.getParent)
```

Comment:
Would this return `false` if the folder already exists? If not should we check the result and fail if it can't be created?